plugin.video.snnow/addon.xml

```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="UTF-8" standalone="yes"?>
-<addon id="plugin.video.snnow" name="Sportsnet Now" version="1.0.9" provider-name="Micah Galizia">
+<addon id="plugin.video.snnow" name="Sportsnet Now" version="1.0.10" provider-name="Micah Galizia">
   <requires>
     <import addon="xbmc.python"   version="2.1.0"/>
```

Comment:
Can you update the python api version as well? It should be `2.20.0` for Isengard